src/analysis/retail/mage/frost/talents/Cryopathy.tsx

```diff
@@ -0,0 +1,54 @@
+import Analyzer, { Options } from 'parser/core/Analyzer';
+import TALENTS from 'common/TALENTS/mage';
+import RayOfFrost from 'analysis/retail/mage/frost/talents/RayOfFrost';
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
+import { formatNumber } from 'common/format';
+
+const STACK_DMG_INCREASE = 0.05;
+
+export default class Cryopathy extends Analyzer {
+  static dependencies = {
+    rayOfFrost: RayOfFrost,
+  };
+
+  protected rayOfFrost!: RayOfFrost;
+
+  constructor(options: Options) {
+    super(options);
+    this.active = this.selectedCombatant.hasTalent(TALENTS.CRYOPATHY_TALENT);
+  }
+
+  statistic(): React.ReactNode {
+    let cryopathyTotalDamage = 0;
+
+    this.rayOfFrost.rayOfFrost.forEach((ray) => {
+      if (typeof ray.damage !== 'undefined') {
+        const cryopathyStacks = this.selectedCombatant.getBuffStacks(
+          SPELLS.CRYOPATHY_BUFF.id,
+          ray.timestamp,
+        );
+        const cryopathyAmount =
+          ray.damage.reduce((totalAmount, damage) => (totalAmount += damage.amount), 0) *
+          STACK_DMG_INCREASE *
+          cryopathyStacks;
+        cryopathyTotalDamage += cryopathyAmount;
```

Comment:
Is it intentional you aren't including `damage.absorbed` here?

Reply:
No, I didn't know that it existed. I'll check on some Tindral log to check the behaviour. Thanks for pointing it out

Reply:
same log than above, with absorbed 👍 

![image](https://github.com/WoWAnalyzer/WoWAnalyzer/assets/32552126/70c7a459-62dc-4a48-b4e6-6e361a080324)